src/runtime_src/xrt/device/hal.h

```diff
@@ -603,6 +603,12 @@ class device
     return operations_result<size_t>();
   }
 
+  virtual operations_result<std::string>
+  getSysfsPath(const std::string subdev, const std::string entry)
```

Comment:
Please pass by reference, e.g. like you do in device.h

Reply:
fixed in new push